src/server/typingsCache.ts

```diff
@@ -157,7 +157,8 @@ export class TypingsCache {
     }
 
     onProjectClosed(project: Project) {
-        this.perProjectCache.delete(project.getProjectName());
-        this.installer.onProjectClosed(project);
+        if (this.perProjectCache.delete(project.getProjectName())) {

```

Comment:
This is the fix for unnecessary queueing of closeProject request

Reply:
Eg tests in unittests\tsserver\autoImportProvider would fail by showing creating TI when the project is closed that does not enable TI